scss/_patterns_media-object.scss

```diff
@@ -12,6 +12,13 @@
 }
 
 @mixin vf-p-media-object {
+  %media-image {
+    border-radius: 4px;
+    height: 3.75rem;
+    margin-right: 1rem;
```

Comment:
Can you use [spacing variables](https://github.com/vanilla-framework/vanilla-framework/blob/develop/scss/_settings_spacing.scss) here, please?